src/pre.js

```diff
@@ -98,13 +101,15 @@ var glpkPromise = new Promise(function (resolve) {
 							time: 0,
 							result: {
 								vars: {},
+								cons: {},
 								z: null,
 								status: 1
 							}
 						},
 						start = new Date().getTime();
-
-					solve(P, ret.result, msg_lev);
+					
+					const settings = lp.settings
```

Comment:
I don't think we should pass the settings as part of the lp object, instead we should change the function `'solve': function (lp, msg_lev) {` from line 93 to be `'solve': function (lp, options) {`